src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -262,7 +261,7 @@ export async function launchGame(
         logFile: lastPlayLogFileLocation(appName),
         logMessagePrefix: LogPrefix.Backend,
         onOutput: (output) => {
-          if (!logsDisabled) appendGamePlayLog(gameInfo, output)
```

Comment:
probably should add that
```
  if (!gameSettings.verboseLogs) {
    appendGamePlayLog(
      gameInfo,
      'IMPORTANT: Logs are disabled. Enable verbose logs before reporting an issue.'
    )
  }
  ```
  here as well ?

Reply:
hmmm looks like we have some issues with how we log things with sideloaded games, for native games the logs are just broken (we log the wrong thing in the wrong file and no system info / config)

I'm trying to fix that, but feel that should be its own PR to limit the scope of this one